src/pages/HomePage.vue

```diff
@@ -14,18 +14,46 @@
       <div class="fit row justify-center">
         <ProjectGrid
           class="col-md-8"
-          :projects="getProjects()"
+          :projects="projects"
         />
       </div>
     </div>
     <import-project-dialog/>
+    <delete-project-dialog/>
   </q-page>
 </template>
 
 <script setup>
 import ProjectGrid from 'src/components/grid/ProjectGrid';
 import { getProjects } from 'src/composables/Project';
 import ImportProjectDialog from 'components/dialog/ImportProjectDialog';
+import DeleteProjectDialog from 'components/dialog/DeleteProjectDialog';
+import ProjectEvent from 'src/composables/events/ProjectEvent';
+import {
+  ref,
+  onMounted,
+  onUnmounted,
+} from 'vue';
+
+const projects = ref(getProjects());
```

Comment:
suggestion 
`const projects = ref({});`

Reply:
don't think so, doc of the `getProjects()`  function says
`* @return {Object} Object that contains all project ids as keys and associated projects as values.`

Reply:
done